wmn-data.json

```diff
@@ -2931,6 +2931,16 @@
         "known" : ["test"],
         "cat" : "gaming"
        },
+       {
+        "name": "komi",
+        "uri_check": "https://{account}.komi.io",
+        "e_code": 200,
+        "e_string": "Privacy Preferences",
+        "m_string": "This page is not currently available",

```

Comment:
Same for this. In the page source and response I don't see this as it is returned later after some other webcalls.